mainForm.Designer.cs

```diff
@@ -45,6 +45,7 @@ private void InitializeComponent()
             this.trackBarKillerDelay = new System.Windows.Forms.TrackBar();
             this.labelKillerDelay = new System.Windows.Forms.Label();
             this.labelKillerDelayMs = new System.Windows.Forms.Label();
+            this.checkBox1 = new System.Windows.Forms.CheckBox();

```

Comment:
Please give `checkBox1` a better name.

Reply:
Lol, I will see, sorry for the late response btw